requirements.txt

```diff
@@ -4,6 +4,6 @@ galaxy_importer>=0.4.1
 GitPython>=3.1.24
 jsonschema>=3.0
 packaging
-pulpcore>=3.16
+pulpcore>=3.18
```

Comment:
Can you pinpoint a change that makes this necessary?

Reply:
It's because of this: https://github.com/pulp/pulp_ansible/pull/849/files#diff-e11a4087ff75182a1c5c58b3812e77def3dccfe96c12e2f6261760225d5c41d2R470. We're using the new content redirect guard.

We don't _technically_ need to bump the version the way that it's written now, but this should probably be re-written to import `RedirectContentGuard` and then compare using `if guard.TYPE == RedirectContentGuard.TYPE` instead of `if guard.TYPE == "content_redirect"`

Reply:
@mdellweg lmk if we should leave it at 3.16 and keep the content guard comparison as is, or bump to 3.18 and import the content guard

Reply:
Thank you. That is exactly the information i wanted to hear (i forgot about).
Keep it exactly the way it is.